server/schedule/placement/rule_manager.go

```diff
@@ -428,13 +431,15 @@ func (m *RuleManager) GetRuleGroups() []*RuleGroup {
 }
 
 // SetRuleGroup updates a RuleGroup.
+// TODO update ruleGroup after set rule/rules
```

Comment:
em, why need to update ruleGroup?

Reply:
For example, If a new rule with a new groupID is created(set), do we need to update groups in rule_manager?

Reply:
If the group does not exist, it will use default configuration so I suppose it is unnecessary to persist group config.

Reply:
got it.